src/main/resources/edu/cornell/kfs/module/cg/cu-ojb-cg.xml

```diff
@@ -14,6 +14,132 @@
+                              auto-update="none" auto-delete="none" proxy="true">
+            <foreignkey field-ref="awardPurposeCode"/>
+        </reference-descriptor>
+        <collection-descriptor name="awardProjectDirectors" element-class-ref="org.kuali.kfs.module.cg.businessobject.AwardProjectDirector"
+                               collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" proxy="true" auto-retrieve="true" auto-update="object" auto-delete="object">
+            <orderby name="principalId" sort="ASC"/>
+            <inverse-foreignkey field-ref="proposalNumber"/>
+        </collection-descriptor>
+        <collection-descriptor name="awardFundManagers" element-class-ref="org.kuali.kfs.module.cg.businessobject.AwardFundManager"
+                               collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" proxy="true" auto-retrieve="true" auto-update="object" auto-delete="object">
+            <orderby name="principalId" sort="ASC"/>
+            <inverse-foreignkey field-ref="proposalNumber"/>
+        </collection-descriptor>
+        <collection-descriptor name="awardAccounts" element-class-ref="org.kuali.kfs.module.cg.businessobject.AwardAccount"
+                               collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" proxy="true" auto-retrieve="true" auto-update="object" auto-delete="object">
+            <orderby name="chartOfAccountsCode" sort="ASC"/>
+            <orderby name="accountNumber" sort="ASC"/>
+            <inverse-foreignkey field-ref="proposalNumber"/>
+        </collection-descriptor>
+        <collection-descriptor name="awardSubcontractors" element-class-ref="org.kuali.kfs.module.cg.businessobject.AwardSubcontractor"
+                               collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" proxy="true" auto-retrieve="true" auto-update="object" auto-delete="object">
+            <orderby name="awardSubcontractorAmendmentNumber" sort="ASC"/>
+            <orderby name="awardSubcontractorNumber" sort="ASC"/>
+            <orderby name="subcontractorNumber" sort="ASC"/>
+            <inverse-foreignkey field-ref="proposalNumber"/>
+        </collection-descriptor>
+        <collection-descriptor name="awardOrganizations" element-class-ref="org.kuali.kfs.module.cg.businessobject.AwardOrganization"
+                               collection-class="org.apache.ojb.broker.util.collections.ManageableArrayList" proxy="true" auto-retrieve="true" auto-update="object" auto-delete="object">
+            <orderby name="chartOfAccountsCode" sort="ASC"/>
+            <orderby name="organizationCode" sort="ASC"/>
+            <inverse-foreignkey field-ref="proposalNumber"/>
+        </collection-descriptor>
+    </class-descriptor>
+    
     <class-descriptor class="edu.cornell.kfs.module.cg.businessobject.CuAward" table="CG_AWD_T">
```

Comment:
Should CuAward still be mapped here?

Reply:
Yes, we still need the CuAward mapping since it includes column mappings that are not in the Award OJB XML, in addition to adding a mapping for the extended attribute.

Reply:
sounds good